frameworks/Rust/xitca-web/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "xitca-web"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+xitca-http = { git = "https://github.com/fakeshadow/xitca-web.git", branch = "tfb" }
```

Comment:
Can you use the `rev` property here to lock all these down to specific commits? We want to make sure these aren't being updated without PRs here. Other than that, this looks ready to go.

Reply:
Done.